src/partner/components/CorporatePartnerList.tsx

```diff
@@ -48,7 +48,7 @@ const CorpotatePartnerList = () => {
             <ActionItem
               key={`action-${type}-${row.original.code}`}
               type={type}
-              onClick={() => navigate(paths.catalogs.buildPath(row.original.code))}
+              onClick={() => navigate(paths.catalogs.buildPath(row.original.id))}
```

Comment:
[to-do] Here again we need to use some identifier as verbose as possible for the final user,  that is the reason I suggest code instead of Id, however this is still an assumption to validate.